Dockerfile

```diff
@@ -5,16 +5,16 @@ ENV LANG C.UTF-8
 WORKDIR /srv
 
 # System dependencies
-RUN apt-get update && apt-get install --yes python3-pip
+RUN apt-get update && apt-get install --no-install-recommends --yes python3 python3-setuptools python3-pip
```

Comment:
You did `apt-get update -y` on the maas.io dockerfile.

Reply:
Fixed